packages/medusa/src/services/totals.ts

```diff
@@ -89,22 +113,104 @@ class TotalsService extends BaseService {
    * @param {Cart | Object} object - cart or order to calculate subtotal for
    * @return {int} tax total
    */
-  getTaxTotal(object) {
-    const subtotal = this.getSubtotal(object)
-    const shippingTotal = this.getShippingTotal(object)
-    const discountTotal = this.getDiscountTotal(object)
-    const giftCardTotal = this.getGiftCardTotal(object)
-    const tax_rate =
-      typeof object.tax_rate !== "undefined"
-        ? object.tax_rate
-        : object.region.tax_rate
-    return this.rounded(
-      (subtotal - discountTotal - giftCardTotal + shippingTotal) *
-        (tax_rate / 100)
+  async getTaxTotal(object: Cart | Order): Promise<number> {
+    // const subtotal = this.getSubtotal(object)
```

Comment:
if these are not used they should be removed disregard if they are to be used after the merge to tax-api 😄